test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -484,6 +467,43 @@ func (data *testData) assignIP(clientset *kubernetes.Clientset) error {
 	return nil
 }
 
+func (data *testData) assertPodAnnotation(t *testing.T, pods []*testPodInfo, clientset *kubernetes.Clientset) error {
+	namespace := data.e2eTestData.GetTestNamespace()
+	assertNetworkStatusContains := func(networkStatus []v1.NetworkStatus, expectNetworkStatus v1.NetworkStatus) bool {
+		for _, status := range networkStatus {
+			if status.Name == expectNetworkStatus.Name && status.Interface == expectNetworkStatus.Interface {
+				return true
+			}
+		}
+		return false
+	}
```

Comment:
This only validates the interface exist but not other fields, why doesn't it validate the whole actualAnnotation equals to expectedAnnotation?

Reply:
Done.